src/models/network.js

```diff
@@ -111,8 +111,11 @@ Network.prototype.getNetworkStatus = function() {
 		const transport = this.irc.connection.transport;
 
 		if (transport.socket) {
+			const isLocalhost = transport.socket.remoteAddress === "127.0.0.1";
```

Comment:
Shouldn't this also be true for `::1` (IPv6)?

Reply:
What about `localhost`, is that considered a valid `remoteAddr`?

Reply:
This is after DNS resolution

Reply:
And what about IPv6?

Reply:
I haven't gotten it to resolve localhost into an IPv6 addr myself, so I don't know which form exactly it will use.

Reply:
:shipit: